index.js

```diff
@@ -78,7 +78,10 @@ module.exports = function(source, map) {
 
 		callback(null, js.code, js.map);
 	}, err => {
-		this.addDependency(err.file);
+		const file = err.filename || err.file;
```

Comment:
this statement favors the `err.filename` value, is that intended?

Reply:
It wasn't accidental, but it is arbitrary. I don't have a reason to favor either over the other.

Reply:
Revisiting this, it occurs to me that one obvious reason to prefer `err.file` is that it maintains the status quo if there's an error value out there which has both properties. I'll push a change to switch the preference.

Reply:
https://github.com/sveltejs/svelte/blob/dafbdc286eef3de2243088a9a826e6899e20465c/src/compiler/utils/error.ts#L9
The property of CompileError is `filename`, not `file`. Checking is good, but I don't think `file` should be cared.

Reply:
It's been a while now, but I think I believed other error types were reachable here.